influxdb3/point_values.go

```diff
@@ -70,7 +70,7 @@ func (pv *PointValues) SetTimestamp(timestamp time.Time) *PointValues {
 // SetTag sets a tag value and returns the modified PointValues
 func (pv *PointValues) SetTag(name string, value string) *PointValues {
 	if value == "" {
-		fmt.Printf("Empty tags has no effect, tag [%s], measurement [%s]\n", name, pv.MeasurementName)
+		logger.Debug(fmt.Sprintf("Empty tags has no effect, tag [%s], measurement [%s]", name, pv.MeasurementName))
```

Comment:
What about keep it super simple and use this:
```suggestion
		slog.Debug(fmt.Sprintf("Empty tags has no effect, tag [%s], measurement [%s]", name, pv.MeasurementName))
```

The user can configure logger by:

```golang
// enable debug logging
opts := &slog.HandlerOptions{
        Level: slog.LevelDebug,
}

logger := slog.New(slog.NewJSONHandler(os.Stdout, opts))

slog.SetDefault(logger)
```